installation.md

```diff
@@ -0,0 +1,31 @@
+# Installing Leapp tool
+
+## Fedora 26 and 27
+
+	```shell
+	sudo dnf install dnf-plugins-core -y
+	sudo dnf copr enable @leapp/leapp-devel -y
+	sudo dnf install leapp -y
+	```
+
+## CentOS 7
+
+	```shell
+	sudo yum install yum-utils -y
```

Comment:
Move -y parameter behind the positional install options. It makes it more readable.